analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/input/VersionControlledFile.kt

```diff
@@ -47,8 +45,7 @@ class VersionControlledFile internal constructor(
     private fun registerModification(modification: Modification) {
         val type = modification.type
         when (type) {
-            Modification.Type.DELETE -> markedDeleted = true
-            Modification.Type.RENAME -> filename = modification.oldFilename
+            Modification.Type.RENAME -> filename = modification.currentFilename
```

Comment:
We might want to make filename public, since all other modification logic is already in the other file. That way we do not have to check this again and are able to remove the whole registerModification function.

Reply:
RIght now registerModification() handles registering the different metrics as well, should we move the metric map as well?
Or make it public, to allow for easier access over VersionControlledFile?

Reply:
AFAIK they are handled in `registerCommit`?

Reply:
I'm not sure what you mean?
Each metric is registered per modification in VersionControlledFile

Reply:
The private `registerModification` function does not register the metrics. That is done in `registerCommit`. I would only want to remove `registerModification`.

Reply:
Yeah, I think I missunderstood
I moved the registration to  `registerCommit` and handled renaming in the other file